### PR TITLE
Update airport.md

### PR DESCRIPTION
**Pull request by ot0tot**
Recommend MAVLink instead of Airport for MAVLink use. Make description of Airport more general than just telemetry.

### CODE REVIEW

File: docs/software/airport.md [hunk 1]
```diff
@@ -5,10 +5,13 @@ description: ExpressLRS can be configured as a bi-directional transparent serial
...
+	
 ## Description
 
 AirPort provides a firmware option that allows you to turn a regular ExpressLRS transmitter and receiver pair into a bi-directional transparent serial data link, over the air.
-The TX module is intended to connect via USB to a laptop, and the RX connects to a free UART on your FC, as per usual. This then allows serial data comms between the peers, in any protocol you wish to use, for example, MAVLink (ArduPilot), MSP (Betaflight and INAV), or any other telemetry protocol you might wish to use. This may also be useful for wireless ground comms, for example: transmitting CRSF from your handset to an antenna tracker.
+A TX module can connect via USB to a laptop, and RX modules (or those flashed using RX-as-TX) can connect to a free UART on your device, as per usual. This allows for a wireless serial data link between TX and RX, as if they were connected directly via serial. Note that due to the packet size and sensitive LoRa modulation parameters used by ELRS, the [data rate](#data-rate-vs-baud-rate) is relatively slow. Also, as the protocol has no knowledge of the data being transported, there is no ACK or retransmission which may result in dropped packets.
```
Comment (deadbytefpv) [inline docs/software/airport.md:14]: "RX modules (or those flashed using RX-as-TX)"
- isn't this wrong?
Don't you mean to say:
"A TX module (or an Receiver converted to a TX module) can connect via USB to a laptop, RX modules can connect to a free UART on your device.."

Might also be wise to retain some of the use-cases for this, like the ground station or antenna tracker.

Comment (ot0tot) [reply docs/software/airport.md:14]: Meaning a TX module can connect to a computer via USB (since they include a UART-USB chip), while receivers or RX-as-TX can connect via UART to any device. i.e. Airport does not require one side to be a computer, you can use it as a general serial link between any two devices. 

Do we need to include specific examples, especially since there are better ways to do them now? For example, an antenna tracker is probably better served by the Sentinel tracker support, ESPnow, or relaying Mavlink over WiFi.

Comment (deadbytefpv) [reply docs/software/airport.md:14]: I feel like there will be some confusion with the wording. 
Although, arguably, any wording used will be confusing to the easily-confused.

It's best to include all the info to the point of being pedantic, imho, as much as we can, without it being too wordy and ending up a wall of text.

Comment (ot0tot) [reply docs/software/airport.md:14]: Do you have a suggestion on what other info should be included?

Comment (deadbytefpv) [reply docs/software/airport.md:14]: Hardware Set ups (TX module to usb, receiver on a different uart; receiver converted to tx on arduino, another receiver on another arduino, TX module on usb, TX module converted to RX on a drone)

This should help better explain that opening paragraph, I think. Doesn't have to be that detailed. Just enough to get a better picture of the hardware topology.

Then maybe add the section for the better options for AATs, etc.

Comment (ot0tot) [reply docs/software/airport.md:14]: > Hardware Set ups (TX module to usb, receiver on a different uart; receiver converted to tx on arduino, another receiver on another arduino, TX module on usb, TX module converted to RX on a drone)

Isn't this all there already? 

```"A TX module can connect via USB to a laptop, and RX modules (or those flashed using RX-as-TX) can connect to a free UART on your device, as per usual. This allows for a wireless serial data link between TX and RX, as if they were connected directly via serial."```

**Review by deadbytefpv — APPROVED** [review]
